unison-cli/src/Unison/Codebase/Editor/Propagate.hs

```diff
@@ -703,3 +714,7 @@ computeDirty getDependents patch shouldUpdate =
 
     edited :: Set Reference
     edited = R.dom (Patch._termEdits patch) <> R.dom (Patch._typeEdits patch)
+
+nameNotInLibNamespace :: Name -> Bool
```

Comment:
The readme shouldn't be a the top level; it will come from myproject.main.Readme; we could start doing that now @hojberg wdyt

We should make doc-writing work without any libs, e.g. we could pre-install an anonymous version of base and bring it into scope if there's no lib called `base`. If you were able to install UCM, you should be able to go offline, create a new codebase and still write docs.